packages/-build-infra/src/utilities/edition-detector.js

```diff
@@ -5,7 +5,7 @@ const path = require('path');
 module.exports = function(blueprint) {
   blueprint.filesPath = function() {
     let rootPath = process.env.EMBER_VERSION === 'OCTANE' ? 'native-files' : 'files';
-    return path.join(this.path, rootPath);
+    return path.join(blueprint.blueprintPath, rootPath);
```

Comment:
I’d vaguely prefer to use `root` vs `blueprintPath`. On addons `root` has the same meaning as this property (it’s the directory that the addon itself is in), and `blueprintPath` is actually a hook (aka method) that returns the parent directory of all the addons’ blueprints. 

Tldr; I’d like to avoid the confusion of using a common name (`blueprintPath`) in a different / confusing way.

Reply:
It's not the directory the addon itself is in, it's where the blueprint is, i.e. root/blueprints/<blueprint_name>

Reply:
Yep, I know 😃. My point is that I’d like `this.root` to always mean “this things root path” whether that thing is an addon, a blueprint, a command, etc. 

Using the name `blueprintPath` here is confusing because the same name on an addon means a **completely** different thing.

Reply:
Ok, changing it.